server/core/app.ts

```diff
@@ -14,9 +14,8 @@ import {
 } from 'logging';
 
 export const returnExpress: (
-  serverName: string,
   getConfig: () => serverConfig
-) => express.Application = (serverName, getConfig) => {
+) => express.Application = (getConfig) => {
```

Comment:
Why no server name?

Reply:
It isnt needed! As there is another parameter the linter didnt complain, but this was never used